packages/passport/sdk/src/Passport.int.test.ts

```diff
@@ -180,7 +179,7 @@ describe('Passport', () => {
           });
 
           expect(accounts).toEqual([mockUserZkEvm.zkEvm.ethAddress]);
-          expect(mockGetUser).toHaveBeenCalledTimes(1);
+          expect(mockGetUser).toHaveBeenCalledTimes(2);
```

Comment:
Reading through the code a bit more, it looks like it's because of the added call in the constructor. Do we have a test specifically for checking this call when a new class is initialised? 🤔

Reply:
yep that is indeed why, I guess one could argue its not clear to begin with why we are asserting the `getUser` call here, so I don't think we need any explicit callouts for this. 

I'll add an explicit test for this case, ty!